t/kubernetes/discovery/kubernetes3.t

```diff
@@ -187,6 +187,13 @@ _EOC_
                         core.log.err("operator k8s cluster error: ", err)
                         return 500
                     end
+
+                    ngx.sleep(1)
+
+                    local k8s = require("apisix.discovery.kubernetes")
+                    local data = k8s.dump_data()
+                    ngx.say(core.json.encode(data,true))
+
```

Comment:
please also add another test that shows usage of querying endpoints using the control API

Reply:
I will add later